### ISSUE TITLE
Add RemoteTech support

### ISSUE THREAD

**Issue by GER-Space** [GER-Space/Kerbal-Konstructs_1.2#3]
The Groundstations could be used as RemoteTech groundstations. This could replace the Downlink 

Check, if KerbNet can also provide additional stations.

**Comment by Kerbas-ad-astra** [comment]
KerbNet is the biome/altitude/anomaly map, which might be cool to add stations and bases to, but the communication system is CommNet.  I'm not sure what the distinction is between the [CommNet](https://kerbalspaceprogram.com/api/class_comm_net_1_1_comm_network.html) and [CommNetNetwork](https://kerbalspaceprogram.com/api/class_comm_net_1_1_comm_net_network.html) classes ([CommNetScenario](https://kerbalspaceprogram.com/api/class_comm_net_1_1_comm_net_scenario.html) has-a CommNetNetwork which has-a CommNet?), but they both have methods for adding [CommNodes](https://kerbalspaceprogram.com/api/class_comm_net_1_1_comm_node.html).

**Comment by GER-Space** [comment]
You are right... I should know better, because I already added Kerbnet and comment to a kos mod... 
I shouldn't code or open issues , when I'm tired.

I hope, that the ishome flag defines a groundstation.
